src/cli.js

```diff
@@ -422,7 +422,7 @@ function collect(fp, files, ignores, ext) {
 
   if (!shjs.test("-e", fp)) {
     cli.error("Can't open " + fp);
-    return;
+    exports.exit(1);
```

Comment:
@caitp I didn't consider that. I think a non-zero exit code for faulty file
references is common behavior for many tools (at least, those that are not
expected to create non-existent files):

    $ cat foobar; echo $?
    cat: foobar: No such file or directory
    1
    $ less foobar; echo $?
    foobar: No such file or directory
    1
    $ ls foobar; echo $?
    ls: cannot access foobar: No such file or directory
    2
    $ d8 foobar; echo $?
    Error reading 'foobar'
    1

There may be a case to be made for backwards compatibility, but the current
behavior is unspecified and (I think) surprising. Were you suggesting this to
avoid disrupting existing build processes?